live-site/components/welcome-section/WelcomeSection.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import PrimaryButton from '../../../shared-ui/components/primary-button/PrimaryButton';
+import { StyledSectionHeader } from '../coming-up/ComingUp.styles';
+import {
+  StyledButtonContainer,
+  StyledRaceContainer,
+  StyledWelcomeSectionContainer,
+  StyledWelcomeSectionContent,
+  StyledWelcomeText
+} from './WelcomeSection.styles';
+
+const WelcomeSection: React.FC = () => {
+  return (
+    <StyledWelcomeSectionContainer>
+      <StyledWelcomeSectionContent>
+        <StyledSectionHeader>Welcome to Hackbeanpot 2023!</StyledSectionHeader>
+        <StyledWelcomeText>
+          {`At HackBeanpot 2023, we aim to create a welcoming environment of ‘explorers’ focused on creativity, learning, and community connection.\n\nHackers can expect to put their resourcefulness to the test during their hacking missions. Whether you’re a seasoned hackathon-goer, an ‘I-have-never-written-a-line-of-code beginner, or someone in between, we’re excited for you to embark on this adventure with us!\n
+            \nLearn more at `}
+          <u>www.hackbeanpot.com</u>
+        </StyledWelcomeText>
+        <StyledButtonContainer>
+          <PrimaryButton btnText="Join our Slack" btnLink=""></PrimaryButton>
```

Comment:
I think this button needs to get smaller on mobile (see add to calendar button in next section)

Reply:
I just checked and I realized its because Mike is adding a scale: 0.7 to the button for the add to calendar. But I think that button should be fine at the regular size? And if not we should just make all of them smaller not just that one LOL. 

What do ya'll think? Based on the designs does mike's button look like the correct size or does the normal one look like the correct size?

Reply:
@surbhi-gulati @Dhivas-Sugumar

Reply:
@jzhzhang66 i think it looks correct, @surbhi-gulati thoughts?

Reply:
Okay cool then would you mind removing the scale: 0.7 that Mike added to his button? So they're all the same size hahah